src/pages/moonsil/Login/Login.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import './Login.scss'; //sass랑 css파일은 이름을 붙여주지 않아도 된다.
+import { withRouter } from 'react-router-dom';
+
+class Login extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      id: '',
+      pw: '',
+      idCondition: '',
+      pwCondition: '',
+    };
+  }
+
+  handleIdInput = e => {
+    this.setState({
+      id: e.target.value,
+      idCondition: this.state.id.indexOf('@') !== -1,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      pw: e.target.value,
+      pwCondition: this.state.pw.length >= 5,
+    });
+  };
+
+  goToMain = () => {
+    this.props.history.push('/Main-Moonsil');
+  };
+
+  render() {
+    return (
+      <div className="Login">
```

Comment:
```suggestion
  render() {
    let idValidation = this.state.id.indexOf("@");
    let pwValidation = this.state.pw.length >= 5
    return (
      <div className="Login">
```
setState 후에는 렌더 함수가 실행된다는 부분 명심하세요!!
그렇다면 유효성 부분을 state 넣지 않고 변수로 사용할 수도 있습니다 :)

Reply:
앗 그리고 className="login" 으로 변경해주세요. camelCase!!